class_classroom.rb

```diff
@@ -0,0 +1,15 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  # Method to add a student to the classroom
+  def add_student(age, name)
+    student = Student.new(age, self, name)
```

Comment:
- [ ] Great job for pushing the student here but I think when assigning you should assign the classroom itself like shown in the [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) so kindly fix it here ⭐💯✅✨

> You could add it like this 🔥


```
  def add_animal(animal)
    @animals.push(animal)
    animal.owner = self
  end
```